src/ngx_http_lua_headers.c

```diff
@@ -1134,6 +1135,22 @@ ngx_http_lua_ffi_get_resp_header(ngx_http_request_t *r,
 
         break;
 
+    case 13:
+        last_modified = r->headers_out.last_modified_time;
+        if (last_modified >= 0) {
+            p = ngx_palloc(r->pool, sizeof("Mon, 28 Sep 1970 06:00:00 GMT"));
+            if (p == NULL) {
+                *errmsg = "no memory";
+                return NGX_ERROR;
+            }
+
+            values[0].data = p;
+            values[0].len = ngx_http_time(p, last_modified) - p;
+            return 1;
+        }
```

Comment:
maybe we can return 0 here when the `last_modified < 0`? just for better performance.